### DISCUSSION TITLE
Optional UDP traffic with additional raw (filtered) RDS data

### DISCUSSION THREAD

**Discussion by andimik** [PE5PVB/TEF6686_ESP32#604] [Ideas]
I propose to add an option in the UDP section to include raw (or filtered) RDS in udp packets.

Example:
```
from=TEF_tuner 1;raw=5264 0529 3C3E 5244
```

Why option and not change existing? Because Stationlist won't work when the traffic is too much (and does not understand this command at all).

The use case is connecting multiple TEFs to one PC via WiFi and recording all what they are receiving/scanning. This prevents the usage of multiple `xdr-gtk` sessions.

On client side you can split it to the various TEF by simple Linux commands.

**Comment by PE5PVB** [comment]
I even didn't know stationlist could decode RDS blocks. Is this additional to current rds data or as replacement?

**Comment by andimik** [comment]
Well it's an idea, not a real requirement. Let's discuss if others need it as well.

No, Stationlist cannot decode RDS blocks. The idea is to combine RDS from various sources.

**Reply by PE5PVB** [reply]
I don't understand. Can you give me some examples about your idea?

**Reply by andimik** [reply]
The idea is that we get the raw RDS on multicast, rather than TCP (which requires additional `*F` in order to start).

But maybe I'll be the only one asking for such a feature ...
